package.json

```diff
@@ -1,30 +1,38 @@
 {
-  "name": "contentful",
+"name": "contentful",
   "description": "Client for Contentful's Content Delivery API",
   "version": "1.1.1",
   "homepage": "https://www.contentful.com/developers/documentation/content-delivery-api/",
   "main": "index.js",
+  "browser": "index.es5.js",
   "repository": "git@github.com:contentful/contentful.js.git",
   "author": "Stephan Seidt <stephan@contentful.com>",
   "license": "MIT",
   "dependencies": {
     "axios": "^0.5.2",
-    "contentful-resolve-response": "~0.1.0",
-    "lodash": "~2.4.1",
-    "redefine": "~0.2.0"
+    "contentful-resolve-response": "~0.1.0"
   },
   "devDependencies": {
+    "babel": "^4.7.16",
+    "babelify": "^6.0.1",
     "bluebird": "^2.0.0",
-    "browserify": "~3.20.0",
+    "browserify": "^9.0.6",
     "buster": "~0.7.6",
     "envify": "~1.0.1",
+    "exorcist": "^0.1.6",
+    "jscs": "^1.12.0",
+    "lodash": "~2.4.1",
     "saucie": "^0.1.0",
     "testem": "~0.6.2",
-    "uglify-js": "^2.4.19"
+    "uglifyify": "^3.0.1"
   },
   "scripts": {
+    "jscs": "jscs --esnext index.es6.js",
+    "bower-build": "browserify -s contentful --debug -t [ babelify --sourceMapRelative . ] -t uglifyify index.es6.js | exorcist dist/contentful.min.js.map > dist/contentful.min.js",
+    "prepublish": "babel index.es6.js -s -o index.es5.js",
+    "watch": "babel index.es6.js -s -w -o index.es5.js",
```

Comment:
Why do we have `index.es5.js`, `index.es6.js` and `dist/contentful.min.js` now? Shouldn't `index.es5.js` stay `dist/contentful.js`?